server/migrations/20251105142314-fix-references-linkTextAndValue.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
-      "disclosure-faq": { "author": { "refId": "author", "type": "metadata", "value": "James Scholes", "linkText": "" }, "authorEmail": { "refId": "authorEmail", "type": "metadata", "value": "james@pac.bz", "linkText": "" }, "title": { "refId": "title", "type": "metadata", "value": "Disclosure of Answers to Frequently Asked Questions Example", "linkText": "" }, "reference": { "refId": "reference", "type": "metadata", "value": "reference/2025-8-8_7334/disclosure-faq.html", "linkText": "Test Case Page for Disclosure of Answers to Frequently Asked Questions Example" }, "designPattern": { "refId": "designPattern", "type": "metadata", "value": "https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/", "linkText": "APG Pattern: Disclosure (Show/Hide)" }, "example": { "refId": "example", "type": "metadata", "value": "https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/examples/disclosure-faq/", "linkText": "APG Example: Disclosure (Show/Hide) for Answers to Frequently Asked Questions" }, "aria-expanded": { "refId": "aria-expanded", "type": "aria", "value": "aria-expanded", "linkText": "aria-expanded" }, "button": { "refId": "button", "type": "htmlAam", "value": "button", "linkText": "button" }, "div": { "refId": "div", "type": "htmlAam", "value": "div", "linkText": "div" }, "ul": { "refId": "ul", "type": "htmlAam", "value": "ul", "linkText": "ul" } },
+      "disclosure-faq": { "author": { "refId": "author", "type": "metadata", "value": "James Scholes", "linkText": "" }, "authorEmail": { "refId": "authorEmail", "type": "metadata", "value": "james@pac.bz", "linkText": "" }, "title": { "refId": "title", "type": "metadata", "value": "Disclosure of Answers to Frequently Asked Questions Example", "linkText": "" }, "reference": { "refId": "reference", "type": "metadata", "value": "reference/2025-8-8_7334/disclosure-faq.html", "linkText": "Test Case Page for Disclosure of Answers to Frequently Asked Questions Example" }, "designPattern": { "refId": "designPattern", "type": "metadata", "value": "https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/", "linkText": "APG Pattern: Disclosure (Show/Hide)" }, "example": { "refId": "example", "type": "metadata", "value": "https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/examples/disclosure-faq/", "linkText": "APG Example: Disclosure (Show/Hide) for Answers to Frequently Asked Questions" }, "aria-expanded": { "refId": "aria-expanded", "type": "aria", "value": "aria-expanded", "linkText": "aria-expanded" }, "button": { "refId": "button", "type": "htmlAam", "value": "button", "linkText": "button" }, "div": { "refId": "div", "type": "htmlAam", "value": "div", "linkText": "div" }, "ul": { "refId": "ul", "type": "htmlAam", "value": "ul", "linkText": "ul" }, "generic": { "refId": "generic", "type": "aria", "value": "generic", "linkText": "generic" } },
```

Comment:
Had to account for the deleted generic reference that was removed in [this PR](https://github.com/w3c/aria-at/pull/1336). Otherwise, was experiencing crashes with latest dump